tasks/react/medium/todo-app/src/App.jsx

```diff
@@ -1,15 +1,67 @@
 // React - Medium
 
-import './App.css';
+import React, { useState, useEffect } from "react";
+import "./App.css";
```

Comment:
To-Do application looks to be working fine, nicely done! Although that `React` import is not necessary because it is not used, only useState and useEffect are needed, but otherwise everything looks good.